### ISSUE TITLE
Build error in debian stretch

### ISSUE THREAD

**Issue by aurabindo** [exaile/exaile#454]
I used to build on Debian Stable before, but after a long time I tried today and the build is failing.

```
aj@debaura:/media/aj/Buffer/git-clones/exaile$ make
python2 -m compileall -q xl xlgui
python2 -O -m compileall -q xl xlgui
make -C plugins compile
make[1]: Entering directory '/media/aj/Buffer/git-clones/exaile/plugins'
python2 -m compileall -q .
python2 -O -m compileall -q .
make[1]: Leaving directory '/media/aj/Buffer/git-clones/exaile/plugins'
mkdir -p build
python2 tools/generate-completion.py > build/exaile.bash-completion
( set -o pipefail || true && \
  export LC_ALL=C && cd po && \
  { find ../xl ../xlgui -name "*.py" | sort && \
    find ../data/ui -name "*.ui" | sort && \
    find ../plugins -name "*.py" | sort && \
    find ../plugins -name "*.ui" | sort ; } \
  | xgettext --files-from=- --output=messages.pot --from-code=UTF-8 --add-comments=TRANSLATORS --keyword=N_ && \
  find ../plugins -name PLUGININFO | sort \
  | xgettext --files-from=- --output=messages.pot --from-code=UTF-8 --add-comments=TRANSLATORS --join-existing --language=Python )
/bin/sh: 1: set: Illegal option -o pipefail
Makefile:208: recipe for target 'pot' failed
make: *** [pot] Error 2
```

### Environment

* OS: Debian Stretch, x86_64

* Exaile Version:  git 67a207a6088be8f4308f3bf59e1e214084fb8cd6

**Comment by sjohannes** [comment]
Thanks for the report. I know what's wrong but will need some time to test the solution. For now you can remove the `set -o pipefail || true &&` part in the Makefile.

**Comment by virtuald** [comment]
@sjohannes interesting, I would have thought the PR in #453 would have caught something like this?

**Comment by sjohannes** [comment]
Looks like #453 only does `python -m compileall`; it doesn't run the whole `make all`. Otherwise this would have failed on the Debian and Ubuntu images.

**Comment by virtuald** [comment]
Ah, we should add that then.

**Comment by sjohannes** [comment]
Should be fixed in a6b353c1410116b507d3a0067c075186894f9f7a, feel free to reopen if it still doesn't work.

**Comment by genodeftest** [comment]
> Looks like #453 only does `python -m compileall`; it doesn't run the whole `make all`. Otherwise this would have failed on the Debian and Ubuntu images.

Thanks, fixed in #460.
